netrd/distance/hamming_ipsen_mikhailov.py

```diff
@@ -0,0 +1,228 @@
+"""
+hamming_ipsen_mikhailov.py
+--------------------------
+
+Graph distance based on paper:
+The HIM glocal metric and kernel for network comparison and classification
+Available here:
+https://ieeexplore.ieee.org/abstract/document/7344816
+
+author: Guillaume St-Onge
+email: guillaume.st-onge.4@ulaval.ca
+Submitted as part of the 2019 NetSI Collabathon.
+
+"""
+
+import numpy as np
+import networkx as nx
+from .base import BaseDistance
+from scipy.optimize import fsolve
+from scipy.sparse.csgraph import laplacian
```

Comment:
We've started preferring dense matrices (#46) but it looks like there is a good reason to use a sparse matrix here, since I don't think `laplacian` is implemented in numpy.

@tlarock

Reply:
It is implemented in networkx. It shouldn't matter if it returns a `matrix` as long as we immediately convert it to an array. Or would that still show  warnings?

Reply:
If it throws deprecation warnings, we'll deal with it later.